packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -87,582 +97,842 @@ describe('quantic-generated-answer', () => {
-              Actions.dislikeGeneratedAnswer();
-              Expect.displayFeedbackModal(false);
+              it(
+                'should display feedback buttons',
+                {
+                  retries: 12,
+                },
+                () => {
+                  Expect.displayLikeButton(true);
+                  Expect.displayDislikeButton(true);
+                  Expect.likeButtonIsChecked(false);
+                  Expect.dislikeButtonIsChecked(false);
+
+                  scope('when liking the generated answer', () => {
+                    Actions.likeGeneratedAnswer();
+                    if (analyticsMode === 'next') {
+                      NextAnalyticsExpectations.emitQnaLikeEvent(
+                        {
+                          feedback: {
+                            liked: true,
+                          },
+                          answer: {
+                            id: streamId,
+                            type: answerType,
+                          },
+                        },
+                        exampleTrackingId
+                      );
+                    } else {
+                      Expect.logLikeGeneratedAnswer(streamId);
+                    }
+                    Expect.likeButtonIsChecked(true);
+                    Expect.dislikeButtonIsChecked(false);
+                  });
+
+                  scope('when disliking the generated answer', () => {
+                    Actions.dislikeGeneratedAnswer();
+                    if (analyticsMode === 'next') {
+                      NextAnalyticsExpectations.emitQnaDislikeEvent(
+                        {
+                          feedback: {
+                            liked: false,
+                          },
+                          answer: {
+                            id: streamId,
+                            type: answerType,
+                          },
+                        },
+                        exampleTrackingId
+                      );
+                    } else {
+                      Expect.logDislikeGeneratedAnswer(streamId);
+                    }
+                    Expect.likeButtonIsChecked(false);
+                    Expect.dislikeButtonIsChecked(true);
+                    Expect.displayFeedbackModal(true);
+                  });
+
+                  scope('when closing the feedback modal', () => {
+                    Actions.clickFeedbackCancelButton();
+                    Expect.displayFeedbackModal(false);
+                  });
+
+                  scope('when selecting a feedback option', () => {
+                    Actions.dislikeGeneratedAnswer();
+                    Actions.clickFeedbackOption(
+                      feedbackOptions.indexOf(irrelevantOption)
+                    );
+                    Actions.clickFeedbackSubmitButton();
+                    if (analyticsMode === 'next') {
+                      NextAnalyticsExpectations.emitQnaSubmitNegativeFeedbackEvent(
```

Comment:
```suggestion
                      NextAnalyticsExpectations.emitQnaSubmitFeedbackEvent(
```

same here, everywhere else its `SubmitFeedback` so it would remove the negative, but its up to you